compiler/luci/import/src/Importer.cpp

```diff
@@ -41,7 +42,7 @@ namespace
 {
 
 void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &reader,
-                   loco::Graph *graph)
+                   loco::Graph *graph, bool copy_consts)
```

Comment:
Can you please leave existing `convert_graph` and add a new one with `copy_consts` ?